app/services/sign_in/session_creator.rb

```diff
@@ -19,6 +20,10 @@ def perform
 
     private
 
+    def validate_credential_lock
+      raise Login::Errors::CredentialLockedError, 'Credential is locked' if user_verification.locked
```

Comment:
Can you add a `CredentialLockedError` to the `SignIn` errors definition and then use that? (I just want to keep sign in service errors scoped to `SignIn`)